openedx/core/djangoapps/theming/finders.py

```diff
@@ -0,0 +1,65 @@
+"""
+Static file finders for Django.
+https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-STATICFILES_FINDERS
+Yes, this interface is private and undocumented, but we need to access it anyway.
```

Comment:
Have we asked about making this interface supported? Or what others do about the private interface?